types/tizen-common-web/index.d.ts

```diff
@@ -1,84 +1,14699 @@
+     *
+     * @privilegeLevel public
+     * @privilegeName http://tizen.org/privilege/alarm
+     *
+     * @throw WebAPIException with error type SecurityError, if the application does not have the privilege to call this method.
+     * @throw WebAPIException with error type UnknownError, if the method cannot be completed because of an unknown error.
+     */
+    removeAll(): void;
+    /**
+     * The period of a day.
+     * It defines the number of seconds per day.
+     */
+    readonly PERIOD_DAY: 86400;
+    /**
+     * The period of an hour.
+     * It defines the number of seconds per hour.
+     */
+    readonly PERIOD_HOUR: 3600;
+    /**
+     * The period of a minute.
+     * It defines the number of seconds per minute.
+     */
+    readonly PERIOD_MINUTE: 60;
+    /**
+     * The period of a week.
+     * It defines the number of seconds in a week.
+     */
+    readonly PERIOD_WEEK: 604800;
+}
+/**
+ * The AlarmRelative interface provides the relative alarm, which occurs at a fixed interval in future.
+ *
+ * This alarm triggers after a duration mentioned in the _delay_ attribute from the moment the alarm is added.
+ * If a _period_ is provided, the alarm keeps triggering for the given interval.
+ *
+ * @remark Since Tizen 2.4 behaviour of this alarm has changed. In order to decrease the power consumption,
+ * the operating system decides when this alarm is going to be fired and what is the period between subsequent executions.
+ * It is guaranteed that this alarm will be fired after at least _delay_ seconds.
+ * If _period_ is provided, it will be adjusted by the operating system, however this value will not be lower than 600 seconds.
+ */
+export class AlarmRelative extends Alarm {
+    constructor(delay: number, period?: number | null);
+    /**
+     * An attribute to store the difference in time (in seconds) between when an alarm is added and when it is triggered.
+     *
+     * @remark Since Tizen 2.4 the operating system adjusts this value to decrease the power consumption.
+     */
+    readonly delay: number;
+    /**
+     * An attribute to store the duration in seconds between each trigger of an alarm.
+     * By default, this attribute is set to null, indicating that this alarm does not repeat.
+     *
+     * @remark Since Tizen 2.4 the operating system adjusts this value to decrease the power consumption.
+     */
+    readonly period: number | null;
+    /**
+     * Returns the duration in seconds before the next alarm is triggered.
+     *
+     * If the alarm has expired, this method returns null.
+     *
+     * @returns The duration before the next alarm is triggered.
+     *
+     * @throw WebAPIException with error type UnknownError, if the method cannot be completed because of an unknown error.
+     */
+    getRemainingSeconds(): number | null;
+}
+export interface AlarmRelative {
```

Comment:
Another case of incorrectly declaring the *static* (i.e., constructor) side of the class on an interface that represents the *instance*. Please remove.